src/Uno.UI/UI/Xaml/Media/AcrylicBrush/Android/RealtimeBlurView.Android.cs

```diff
@@ -0,0 +1,413 @@
+		private Paint mPaint;
+		private readonly Rect mRectSrc = new Rect();
+		private readonly Rect mRectDst = new Rect();
+		// mDecorView should be the root view of the activity (even if you are on a different window like a dialog)
+		private View mDecorView;
+		// If the view is on different root view (usually means we are on a PopupWindow),
+		// we need to manually call invalidate() in onPreDraw(), otherwise we will not be able to see the changes
+		private bool mDifferentRoot;
+		private static int RENDERING_COUNT;
+
+		private readonly PreDrawListener _preDrawListener;
+
+		public RealtimeBlurView(Context context) : base(context)
+		{
+			_preDrawListener = new PreDrawListener(this);
+			mBlurImpl = GetBlurImplementation(); // provide your own by override getBlurImpl()			
+			mBlurRadius = TypedValue.ApplyDimension(ComplexUnitType.Dip, 10, context.Resources.DisplayMetrics);
+			//mDownsampleFactor = a.getFloat(R.styleable.RealtimeBlurView_realtimeDownsampleFactor, 4);
+
+			mPaint = new Paint();
+		}
+
+		public void SetOverlayColor(int color, bool invalidate = true)
+		{
+			if (mOverlayColor != color)
+			{
+				mOverlayColor = color;
+				if (invalidate)
+				{
+					Invalidate();
+				}
+			}
+		}
+
+		protected IBlurImpl GetBlurImplementation() => new AndroidStockBlur();
+
+		private class PreDrawListener : Java.Lang.Object, ViewTreeObserver.IOnPreDrawListener
+		{
+			private readonly JniWeakReference<RealtimeBlurView> _weakBlurView;
+
+			public PreDrawListener(RealtimeBlurView blurView)
+			{
+				_weakBlurView = new JniWeakReference<RealtimeBlurView>(blurView);
+			}
+
+			public PreDrawListener(IntPtr handle, JniHandleOwnership transfer)
+				: base(handle, transfer)
+			{
+			}
+
+			public bool OnPreDraw()
+			{
+				if (!_weakBlurView.TryGetTarget(out var blurView))
+				{
+					return false;
+				}
+
+				int[] locations = new int[2];
+				Bitmap oldBmp = blurView.mBlurredBitmap;
+				View decor = blurView.mDecorView;
+				if (decor != null && blurView.IsShown && blurView.Prepare())
+				{
+					bool redrawBitmap = blurView.mBlurredBitmap != oldBmp;
+					oldBmp = null;
+					decor.GetLocationOnScreen(locations);
+					int x = -locations[0];
+					int y = -locations[1];
+
+					blurView.GetLocationOnScreen(locations);
+					x += locations[0];
+					y += locations[1];
+
+					// just erase transparent
+					blurView.mBitmapToBlur.EraseColor(blurView.mOverlayColor & 0xffffff);
+
+					int rc = blurView.mBlurringCanvas.Save();
+					blurView.mIsRendering = true;
+					RENDERING_COUNT++;
+					try
+					{
+						blurView.mBlurringCanvas.Scale(
+							1f * blurView.mBitmapToBlur.Width / blurView.Width,
+							1f * blurView.mBitmapToBlur.Height / blurView.Height);
+						blurView.mBlurringCanvas.Translate(-x, -y);
+						if (decor.Background != null)
+						{
+							decor.Background.Draw(blurView.mBlurringCanvas);
+						}
+						decor.Draw(blurView.mBlurringCanvas);
+					}
+					catch (StopException)
+					{
```

Comment:
Codacy found an issue: [Either remove or fill this block of code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5932087)